Hyper/HUD/Sprites/SpriteSheetMetadata.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Text.Json.Serialization;
+using OpenTK.Mathematics;
+
+namespace Hyper.HUD.Sprites;
+
+public class SpriteSheetMetadata
+{
+    [JsonPropertyName("width")]
+    public int Width { get; set; }
+    
+    [JsonPropertyName("height")]
+    public int Height { get; set; }
+    
+    [JsonPropertyName("items")]
+    public List<Sprite> Items { get; set; }
```

Comment:
Maybe make it `List?` and add a null check that throws `NullArgumentException`? Idk what would be the cleanest solution here

Reply:
added =null!